cloudoperators/juno
# fix(ui) fix complex form story

# Summary

Add decorator with `<PortalProvider />` to make sure Select menu is rendered correctly. Fixes #176 

# Changes Made
see above

# Related Issues



- Issue 1: #176 

# Screenshots (if applicable)

Before: 
<img width="399" alt="Screenshot 2024-08-01 at 12 49 06" src="https://github.com/user-attachments/assets/5a02e0a3-90e3-4286-b1f8-9a2a26ae2398">

After:
<img width="325" alt="Screenshot 2024-08-01 at 12 49 20" src="https://github.com/user-attachments/assets/70277eb1-4372-4484-9d07-5f964fb25079">


# Testing Instructions

Run storybook and check `Forms / Form / Complex Form` story, check visually by opening the Select menu.
